styles/_font.scss

```
@@ -1,6 +1,11 @@
 /*external sources*/
 @import url('https://fonts.googleapis.com/css2?family=Lalezar&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Noto+Sans+KR:wght@100..900&display=swap');
 
 ._font-roboto {
   font-family: Roboto sans-serif;
```

Stylelint (17.6.0)

[error] Expected quotes around "Roboto sans-serif" (font-family-name-quotes)

(font-family-name-quotes)